recipes/sqcircuit/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "SQcircuit" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9627fb5bdce35018559beb5153ba6c17d6e4529c6a4152b1c07665e1e072d21f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - dill >=0.3.4
+    - matplotlib >=3.0.0
+    - numpy >=1.14.2
+    - pip
+    - python
+    - qutip >=4.3.1
+    - scipy >=1.1.0
```

Most of this isn't needed at package build time. Also need a Python version for noarch:
```suggestion
    - pip
    - python >=3.6
```